src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -93,7 +93,7 @@
     <SonarQubeOutputPath Condition=" $(SonarQubeOutputPath) == '' ">$(SonarQubeTempPath)\out</SonarQubeOutputPath>
 
     <!-- Specify the ItemGroups to be analyzed -->
-    <SQAnalysisFileItemTypes Condition=" $(SQAnalysisFileItemTypes) == '' ">AdditionalFiles;AndroidEnvironment;AndroidJavaSource;AndroidResource;Build;ClCompile;ClInclude;CodeAnalysisDictionary;Compile;Content;DeploymentExtensionConfiguration;EmbeddedResource;EntityDeploy;None;PostDeploy;PRIResource;PreDeploy;RefactorLog;Resource;Script;ScriptCode;TypeScriptCompile</SQAnalysisFileItemTypes>
+    <SQAnalysisFileItemTypes Condition=" $(SQAnalysisFileItemTypes) == '' ">AdditionalFiles;AndroidEnvironment;AndroidJavaSource;AndroidResource;Build;ClCompile;ClInclude;CodeAnalysisDictionary;Compile;Content;DeploymentExtensionConfiguration;EmbeddedResource;EntityDeploy;None;PostDeploy;PRIResource;PreDeploy;RefactorLog;Resource;Script;ScriptCode;TypeScriptCompile;$(SQAdditionalAnalysisFileItemTypes)</SQAnalysisFileItemTypes>
```

Comment:
_SQAdditionalAnalysisFileItemTypes_ is a pretty long name, but I can't think of a good alternative, and it should be a pretty uncommon scenario so I think it's fine.

Reply:
I think it's worth adding a comment in the section at the top to explain what this property is for. Most users won't find it on their own, but it is useful to us to document what it is for, and we can also point users to the comment if appropriate.
I was thinking of something like the following:

> Specifying additional types of file to be analysed
> ==================================================
> The Scanner for MSBuild will automatically collect all of the common
> file types that can be analysed by the various Sonar language plugins
> (e.g. C# files, TypeScript, TSQL, C++ include and source files etc).
> The full list of item types that will be included for analysis is 
> specified in $(SQAnalysisFileItemTypes) property below.
> 
> If you have any additional item types that you want to have analyzed,
> you can specify them setting the property $(SQAdditionalAnalysisFileItemTypes)
> in the project being analysed e.g.
> 
>   <SQAdditionalAnalysisFileItemTypes>MyCustomFileType1;MyCustomFileType2</SQAdditionalAnalysisFileItemTypes>
>